src/index.html

```diff
@@ -11,12 +11,116 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500&
+      display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#logo"
+        class="logo"
+      >
+        <figure class="figure__logo-moyo">
+          <img
+            src="/src/images/logo.png"
+            alt="here is the moyo logo image"
```

Comment:
The `alt` attribute value should describe the image content more precisely. Instead of 'here is the moyo logo image', it should succinctly describe what the logo looks like or represents.